src/api/spec/models/project_spec.rb

```diff
@@ -18,6 +18,14 @@
     it { should allow_value("fOO:+-").for(:name) }
   end
 
+  describe ".image_templates" do
+    let(:attribute_type) { AttribType.find_by_namespace_and_name!('OBS', 'ImageTemplates') }
```

Comment:
This line is also repeated https://github.com/openSUSE/open-build-service/pull/2366/files#diff-5ecd9071c17a5617622026024fed5729R401. We could move it up. :wink: